src/pd/client.rs

```diff
@@ -204,6 +204,34 @@ impl PdClient for RpcClient {
         Ok(resp.take_region())
     }
 
+    fn get_region_info(&self, key: &[u8]) -> Result<RegionInfo> {
+        let _timer = PD_REQUEST_HISTOGRAM_VEC
+            .with_label_values(&["get_region"])
```

Comment:
Why not "get_region_info"?

Reply:
Because this function is logically the same as "get_region", just returns a different type.